tests/ClassTest.php

```diff
@@ -1004,6 +1004,18 @@ public function valid(): bool {
                 ',
                 'error_message' => 'MissingTemplateParam',
             ],
+            'cannotNameClassConstantClass' => [
+                '<?php
```

Comment:
this should have an array offset `'code' =>` to respect the format of tests if you're targeting master